certbot-apache/certbot_apache/_internal/override_void.py

```diff
@@ -0,0 +1,23 @@
+""" Distribution specific override class for Arch Linux """
```

Comment:
s/Arch/Void ;)

Reply:
Hello Osiris. I have pushed a commit to fix this. Sorry for the oversight.